plugin-server/src/types.ts

```diff
@@ -178,6 +180,12 @@ export interface Hub extends PluginsServerConfig {
     conversionBufferEnabledTeams: Set<number>
 }
 
+export interface PluginServerCapabilities {
+    ingestion?: boolean
+    pluginScheduledTasks?: boolean
+    jobs?: boolean
```

Comment:
after some further reading I'm wondering if this naming could eventually cause confusion. Reason being that plugins in the ingestion server can still create/trigger jobs. The server just won't run them.

Wondering if we should use `jobsRunner` or `jobQueueConsumer` or something?